packages/manager/modules/adp/src/adp.constants.js

```diff
@@ -0,0 +1,468 @@
+export const ADP_PUBLIC_CLOUD_STATUS = Object.freeze({
+  CREATING: 'creating',
+  DELETED: 'deleted',
+  DELETING: 'deleting',
+  OK: 'ok',
+  SUSPENDED: 'suspended',
+});
+
+export const ADP_NODE_TYPES = Object.freeze({
+  MASTER: 'MASTER',
+  WORKER: 'SLAVE',
+  EDGE: 'EDGE',
+  UTILITY: 'UTILITY',
+  BASTION: 'BASTION',
+});
+
+export const ADP_GUIDE_LINKS = Object.freeze({
+  general_link: 'https://docs.ovh.com/gb/en/analytics/',
+});
+
+export const ADP_CLUSTER_MANAGE = Object.freeze({
+  AMBARI: 'https://knox.serviceName.datalake.ovh/gateway/default/ambari',
+  FREEIPA: 'https://ipa.serviceName.datalake.ovh',
+  RANGER: 'https://knox.serviceName.datalake.ovh/gateway/default/ranger/login.jsp',
+});
+
+export const ADP_NODE_NAMES = ['worker', 'edge', 'master', 'utility', 'bastion'];
+
+export const ADP_CLOUD_CATALOG_NAME = 'cloud';
+
+export const ADP_FLAVOR_TYPES = Object.freeze([
+  {
+    id: 'balanced',
+    types: ['ovh.ceph.eg', 'ovh.ssd.eg'],
+  },
+  {
+    id: 'cpu',
+    types: ['ovh.cpu', 'ovh.ssd.cpu', 'ovh.ceph.hg'],
+  },
+  {
+    id: 'ram',
+    types: ['ovh.ram', 'ovh.ssd.ram'],
+  },
+  {
+    id: 'accelerated',
+    types: ['ovh.ssd.gpu', 'ovh.ssd.gpu2', 'ovh.ssd.gpu3', 'ovh.ssd.fpga2', 'ovh.raid-nvme.t1'],
+  },
+  {
+    id: 'vps',
+    types: ['ovh.vps-ssd'],
+  },
+]);
+
+export const ADP_CREDENTIALS_INFO = Object.freeze({
```

Comment:
Maybe add some `TODO` comment in order to get rid of theses fixtures as soon as possible?

Reply:
This is done. All constants that have to be removed after the APIs are live, are marked with a TODO comment.